deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/wazuh_deployment/AgentDeployment.py

```diff
@@ -101,11 +101,9 @@ def health_check(self):
 
         tasks_list = []
         tasks_list.append(AnsibleTask({'name': 'Extract service status',
-                                       'command': f'{self.install_dir}/bin/wazuh-control status',
+                                       'command': f'{self.install_dir_path}/bin/wazuh-control status',
```

Comment:
```suggestion
                                       'command': f"{self.install_dir_path}/bin/wazuh-control status",
```

Reply:
done in 
https://github.com/wazuh/wazuh-qa/pull/1715/commits/0e86bd224c659e238be3973a4bad443e4cb22c14